WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -1135,7 +1131,7 @@ protected void onResume() {
 
         if (mBottomNav != null) {
             PageType currentPageType = mBottomNav.getCurrentSelectedPage();
-            trackLastVisiblePage(currentPageType, mFirstResume);
+            trackLastVisiblePage(currentPageType);
```

Comment:
⚠️ Something I noticed after these changes is that now the events are tracked when the device rotates, which doesn't seem desirable. 

In case we don't want to track on orientation changes, one idea is to keep some flag and add some logic related to saving and restoring instance state. I didn't double check, but [these seem to be the Lifecycle callbacks order](https://stackoverflow.com/a/44511611/2237911) during orientation change, so we can probably differentiate between:
- Orientation change with app in foreground (`onSaveInstanceState` called before `onStop`) -> should **not** track the event again when reaching this line inside `onResume`;
- App in background and killed by the system to reclaim memory ( `onSaveInstanceState` called after `onStop`) ->  **should** track the event again when reaching this line inside `onResume`.

But this is just something I thought now, there might be other ways of making sure we track only when we want.

Reply:
Good catch, thanks

I'm fixing it

Reply:
Thanks for finding these tracking issues, @daniloercoli 

Since they're not related to this PR, I've created a [new issue](https://github.com/wordpress-mobile/WordPress-Android/issues/20006) to take care of it.